### PR TITLE
Add llm notebooks

### PR DESCRIPTION
**Pull request by lavraicse**
<br>
Add Bart-large-cnn and dolly-v2 serving notebooks.
<br><br><br>

- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb [hunk 1]
```diff
@@ -0,0 +1,373 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb:1]: <p>Explain functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    from google.cloud import aiplatform
</pre><p>redundant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb [hunk 2]
```diff
@@ -0,0 +1,364 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb:1]: <p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    from google.cloud import aiplatform
</pre><p>redundant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb:1]: I see heading for both notebooks:
https://screenshot.googleplex.com/3JTvLb7LMn3Fcxo
https://screenshot.googleplex.com/7g4EZUZnRUPvGqz

Please clarify what am I missing.

File: notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb (hunk 1 above)
Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb:1]: Done.

Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_pytorch_bart_large_cnn.ipynb:1]: Done.

File: notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb (hunk 2 above)
Comment (lavraicse) [reply notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb:1]: Done.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**